src/common/utils/helper.util.ts

```
@@ -0,0 +1,106 @@
+      });
+
+      if (!existing) {
+        return finalAlias;
+      }
+
+      randomNum = Math.floor(Math.random() * 1000); // Generate new random number
+      finalAlias = `${baseAlias}-${randomNum}`;
+    }
+  }
+
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)